menas/src/main/scala/za/co/absa/enceladus/menas/repositories/OozieRepository.scala

```diff
@@ -94,30 +94,50 @@ class OozieRepository @Autowired() (oozieClientRes: Either[OozieConfigurationExc
     this.initializeJars()
   }
 
-  private def initializeJars() {
-    val hdfsStdPath = new Path(s"$enceladusJarLocation$standardizationJarPath")
-    val hdfsConfPath = new Path(s"$enceladusJarLocation$conformanceJarPath")
-    val mavenStdPath = s"$mavenRepoLocation$standardizationJarPath"
-    val mavenConfPath = s"$mavenRepoLocation$conformanceJarPath"
-
-    if (!hadoopFS.exists(hdfsStdPath) || hadoopFS.getStatus(hdfsStdPath).getCapacity == 0) {
-      logger.info(s"Uploading standardization jar from $mavenStdPath to $hdfsStdPath")
-      val resFuture = this.downloadFile(mavenStdPath, hdfsStdPath)
-      resFuture.onSuccess({ case (u: Unit) => logger.info(s"Standardization jar loaded to $hdfsStdPath") })
-      resFuture.onFailure({
-        case (err: Throwable) =>
-          hadoopFS.delete(hdfsStdPath, true)
-      })
+  private def validateProperties(): Boolean = {
+    Seq((oozieScheduleHDFSPath, "za.co.absa.enceladus.menas.oozie.schedule.hdfs.path"),
+      (enceladusJarLocation, "zza.co.absa.enceladus.menas.oozie.enceladusJarLocation"),
+      (standardizationJarPath, "za.co.absa.enceladus.menas.oozie.mavenStandardizationJarLocation"),
+      (conformanceJarPath, "za.co.absa.enceladus.menas.oozie.mavenConformanceJarLocation"),
+      (mavenRepoLocation, "za.co.absa.enceladus.menas.oozie.mavenRepoLocation"),
+      (menasApiURL, "za.co.absa.enceladus.menas.oozie.menasApiURL"),
+      (splineMongoURL, "za.co.absa.enceladus.menas.oozie.splineMongoURL")).map(p => validateProperty(p._1, p._2)).reduce(_ && _)
+  }
+
+  private def validateProperty(prop: String, propName: String): Boolean = {
```

Comment:
I just now realized this method will always return true. @lokm01  what do we do? New issue or reopen this?

Reply:
created #482